app/models/setting/provisioning.rb

```diff
@@ -101,4 +107,9 @@ def self.map_pxe_kind
       yield pxe_kind, templates
     end
   end
+
+  def self.default_excluded_facts(ignored_interfaces = IGNORED_INTERFACES)
+    # By default ignore facts from both forms: "a::b::macvtap01::d::e" and "a => b => macvtap01 => ..."
+    ignored_interfaces.flat_map { |i| ["*::#{i}", i] }
+  end
```

Comment:
I don't understand, the list is enriched with `*::` here, and then in `find_excluded_facts` method, so is it added twice? I prefer just to have it in one place, the latter seems to be a better fit for this I think.